test/csuite/wt6616_checkpoint_oldest_ts/main.c

```diff
@@ -139,6 +139,7 @@ thread_run(void *arg)
     char kname[64], tscfg[64];
```

Comment:
Can you memset `tscfg` buffer this as well?

Reply:
`tscfg` is fine. We only read the part which is written as it is a string.

Reply:
kanme is different as we are treating it as binary data.

Reply:
Yeah, that makes sense. Thank you.